src/test/java/core/basesyntax/service/ReportServiceTest.java

```diff
@@ -0,0 +1,73 @@
+package core.basesyntax.service;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.impl.ReportServiceImpl;
+import org.junit.After;
+import org.junit.Test;
+
+public class ReportServiceTest {
+
+    @After
+    public void afterEachTest() {
+        Storage.fruits.clear();
+    }
+
+    @Test
+    public void reportService_FirstCase_Ok() {
```

Comment:
**for all test method use the name pattern**
teatedMethodName_description_ok/notOk